public/app/features/datasources/DataSourcesList.tsx

```diff
@@ -1,17 +1,16 @@
 // Libraries
 import { css } from '@emotion/css';
-import React, { FC } from 'react';
+import React from 'react';
+import { useSelector } from 'react-redux';
 
 // Types
-import { DataSourceSettings, LayoutMode } from '@grafana/data';
 import { Card, Tag, useStyles } from '@grafana/ui';
+import { StoreState } from 'app/types';
 
-export interface Props {
-  dataSources: DataSourceSettings[];
-  layoutMode: LayoutMode;
-}
+import { getDataSources } from './state/selectors';
 
-export const DataSourcesList: FC<Props> = ({ dataSources, layoutMode }) => {
+export const DataSourcesList = () => {
+  const dataSources = useSelector((state: StoreState) => getDataSources(state.dataSources));
```

Comment:
rather than tying this component to redux, can the container (DataSourcesListPageContent) talk to redux and pass them to this component?

Reply:
Yes, I can do that 👍

Reply:
Fixed in https://github.com/grafana/grafana/pull/51438/commits/cec7bb6f3aeca95a2d4460f365013c0d53682c33.